packages/commonwealth/client/scripts/views/components/proposal_card/helpers.tsx

```diff
@@ -184,13 +184,10 @@ export const getStatusText = (proposal: AnyProposal) => {
   return '';
 };
 
-export const getPrimaryTagText = (proposal: AnyProposal) => [
-  chainEntityTypeToProposalShortName(
+export const getPrimaryTagText = (proposal: AnyProposal) => `
```

Comment:
Does this array -> string change affect anything?

Reply:
It's supposed to return a string, so the only change is to make it pass the type check.

Reply:
(There's no reason for it to return an array as opposed to a string interpolation)